superset/initialization/__init__.py

```diff
@@ -64,6 +65,11 @@ def __init__(self, app: SupersetApp) -> None:
         self.config = app.config
         self.manifest: Dict[Any, Any] = {}
 
+    @deprecated(details="use self.superset_app instead of self.flask_app")  # type: ignore   # pylint: disable=line-too-long
```

Comment:
Why use that decorator comparing to a comment?

Reply:
it providing proper warnings both in documentation and via Python’s warnings system,